src/components/Form.js

```diff
@@ -1,17 +1,19 @@
 import React, { useState } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { add } from '../redux/books/booksSlice';
+import { useDispatch } from 'react-redux';
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
```

Comment:
- [ ] As requested by the previous reviewer please remove the disabling linters. Remember linters are here to help you write a better code. This one can easily be fixed by adding it on dependencies or devDependencies on the package.json whereas the where the other is there  because it is considered bad practice to modify function parameters directly, as it can lead to unexpected behavior and make code harder to reason about. I think the previous reviewer gave you a good hint on how to solve those.

Reply:
@GzimAsani can you show me how to solve that? 😃